src/Dropzone/README.md

```diff
@@ -277,3 +277,44 @@ This example validates that only `400x479` images can be uploaded.
   );
 }
 ```
+
+## Reading file contents into memory
+
+Accepts only .xml files up to a size of 20MB. You can read in the contents of the `File` object into memory. the onProcessUpload prop should retrieve the file Blob from the passed fileData param and pass it into a file reader.
```

Comment:
nit: "the" -> "The" 

nit x2: perhaps wrap `onProcessUpload` and `fileData` with backticks so it gets the code formatting? (and `handleProcessUpload` below)

Reply:
yup yup yup